src/thing.rs

```diff
@@ -19,7 +19,8 @@ use crate::builder::ThingBuilder;
 pub(crate) type MultiLanguage = HashMap<String, String>;
 pub(crate) type DataSchemaMap = HashMap<String, DataSchema>;
 
-pub(crate) const TD_CONTEXT: &str = "https://www.w3.org/2019/wot/td/v1";
+pub const TD_CONTEXT_10: &str = "https://www.w3.org/2019/wot/td/v1.0";
```

Comment:
it is `https://www.w3.org/2019/wot/td/v1`

Reply:
Doh